src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/AssemblyAttr.cs

```diff
@@ -0,0 +1,29 @@
+﻿/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Tizen.MachineLearning.Service, " + PublicKey.Value)]
+
+internal static class PublicKey
```

Comment:
This file makes Tizen.MachineLearning.Service to access internal function of Tizen.MachineLearning.Inference